.github/actions/spelling/excludes.txt

```diff
@@ -7,10 +7,10 @@
 \.editorconfig$
 \.eslintrc$
 \.git
-\.htmlvalidate\.json$
 \.ico$
 \.jfif$
 \.jpg$
+\.json$
```

Comment:
JSON files can have comments, and it would be useful to spell-check only those. Isn't that possible?

Reply:
I was wondering the same thing, but AFAIK "ignore everything in .json files except comments (based on regex match)" is not possible.